ci/setup_env.sh

```diff
@@ -12,41 +12,30 @@ if [[ "$(uname)" == "Linux" && -n "$LC_ALL" ]]; then
     echo
 fi
 
-MINICONDA_DIR="$HOME/miniconda3"
-
-
-if [ -d "$MINICONDA_DIR" ]; then
-    echo
-    echo "rm -rf "$MINICONDA_DIR""
-    rm -rf "$MINICONDA_DIR"
-fi
 
 echo "Install Miniconda"
-UNAME_OS=$(uname)
-if [[ "$UNAME_OS" == 'Linux' ]]; then
+DEFAULT_CONDA_URL="https://repo.continuum.io/miniconda/Miniconda3-latest"
+if [[ "$(uname -m)" == 'aarch64' ]]; then
+    CONDA_URL="https://github.com/conda-forge/miniforge/releases/download/4.8.5-1/Miniforge3-4.8.5-1-Linux-aarch64.sh"
```

Comment:
Could use the latest version `https://github.com/conda-forge/miniforge/releases/download/4.10.1-4/Miniforge3-4.10.1-4-Linux-aarch64.sh`